Makefile

```diff
@@ -94,4 +93,4 @@ docker-down:
 	cd ./app && docker-compose down
 
 cypress-open:
-	cd ./app && npx cypress open
\ No newline at end of file
+	cd ./app && npx cypress open --env CYPRESS_BASE_URL=http://localhost:3000,CYPRESS_RUN_AS_SMOKETEST=false
```

Comment:
Make files can get funny about using cd commands into folders as the root directory never technically changes when running make commands.

Change to 

`	npm exec --prefix ./app cypress open --env CYPRESS_BASE_URL=http://localhost:3000,CYPRESS_RUN_AS_SMOKETEST=false
`